File: examples/scripts/report.py
# Generate a summary report after test execution.
# A real script could POST to Slack, write a file, etc. — kept pure here.
# Pure stdlib — no pip, no virtualenv.

import datetime

timestamp = datetime.datetime.now(datetime.timezone.utc).strftime("%Y-%m-%d %H:%M:%S")

report = "\n".join(
    [
        "===================================",
        "  Nap Test Report",
        f"  {timestamp}",
        "===================================",
        "",
        "  Runner:  Python",
        "  Status:  Complete",
        "===================================",
    ]
)

print(report)
print("[report] Report generated")

File: examples/scripts/report.js
// Generate a summary report after test execution.
// A real script could POST to Slack, write a file, etc. — kept pure here.
// Pure JS — no imports, no npm, no build.

const timestamp = new Date().toISOString().replace("T", " ").slice(0, 19);

const report = [
  "===================================",
  "  Nap Test Report",
  `  ${timestamp}`,
  "===================================",
  "",
  "  Runner:  JavaScript",
  "  Status:  Complete",
  "===================================",
].join("\n");

console.log(report);
console.log("[report] Report generated");
